.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           node_version: ${{ matrix.node_version }}
 
       - name: E2E tests in firefox
-        run: npm run e2e:ff
+        run: npm run e2e:ff && npm run e2e:vue-showcase:ff
```

Comment:
I think we might need to start doing `affected:e2e` commands in these tests. That way only those tests will run that are relevant for the actual changes.

For example if I make a change in an angular library, it will run the showcase e2e tests but not the vue e2e tests because they don't depend on each other. I created #129 to address this later.

Reply:
This is a great suggestion. Thanks